selection_sort.ts

```diff
@@ -0,0 +1,46 @@
+// Genterates an array of random integers
+let random = (function () {
```

Comment:
We can do a pull request to merge this for testing purposes. Maybe create a separate tests.ts file for each algorithm?